openstack/input.yaml

```diff
@@ -1,5 +1,4 @@
-ssh-key-name: some_key
-image-name: "centos-7 x86_64"
```

Comment:
Why were these 2 lines removed? Although this file is overwritten with inpuy.yaml.tmpl but nevertheless, it should be a valid file with inputs.

Reply:
My bad. No idea how it happened. Good catch.